.trunk/.gitignore

```diff
@@ -0,0 +1 @@
+*out
```

Comment:
We don't want a new top-level directory i.e. `trunk`.

All of our `.gitignores` are controlled via the `make` system.